canonical-websites/mir-server.io
# Add Dockerfile for production image

Fixed #2

QA
--

``` bash
docker build --tag mir-server.io --build-arg COMMIT_ID=`git rev-parse HEAD` .
docker run -ti -p 8928:80 mir-server.io
```

Now go to http://127.0.0.1:8928, and http://127.0.0.1:8928/_status/check
